tools/mo/openvino/tools/mo/moc_frontend/extractor.py

```diff
@@ -399,7 +399,7 @@ def convert_params_lists_to_dicts(input_model,
 
         # this cycle adds each unnamed type to dictionary using name from model_inputs
         for idx, node_type in enumerate(input_user_data_types):
-            assert isinstance(node_type, (type, Type)), "Got incorrect format of input types. " \
+            assert isinstance(node_type, (type, np.dtype, Type)), "Got incorrect format of input types. " \
```

Comment:
@rkazants @popovaan please help me to understand, it is bug in the code or wrong error message here?

error says that numpy dtype is expected, but it is not listed in isinstance check, as the result when I use numpy dtype, this function report error. If it is not expected anymore and desired ov.Type, I can switch on it.

Reply:
Great catch! @popovaan, please advice.